client/src/components/CommentButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { FaRegComment } from 'react-icons/Fa';
+import { styled } from 'styled-components';
+
+const CommentButton = () => {
+  return (
+    <Wrapper>
+      <Button />
+    </Wrapper>
+  );
+};
+
+export default CommentButton;
+
+const Wrapper = styled.aside`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
```

Comment:
```suggestion
const Container = styled.aside`
  display: flex;
  flex-direction: column;
  align-items: center;
`;
```
`Container`로 통일하는건 어떨까용? 👀